node/test_force.js

```diff
@@ -270,26 +274,24 @@ function createCompileApp(tmpDir, appType, os) {
 //
 function updatePluginRepo(tmpDir, pluginRepoDir, branch, os) {
     log('Updating cordova plugin at ' + branch, COLOR.green);
-    shelljs.pushd(pluginRepoDir);
-    runProcessThrowError(path.join('tools', 'update.sh') + ' -b ' + branch + ' -o ' + os);
-    shelljs.popd();
+    runProcessThrowError(path.join('tools', 'update.sh') + ' -b ' + branch + ' -o ' + os, pluginRepoDir);
 }
 
 //
 // Update forceios/forcedroid script to use local plugin repo
 //
 function editForceScriptToUseLocalPluginRepo(tmpDir, os) {
     log('Editing  ' + forcePackageNameForOs(os) + '.js to use local cordova plugin', COLOR.green);
-    miscUtils.replaceTextInFile(path.join(tmpDir, 'node_modules', forcePackageNameForOs(os), 'node', forcePackageNameForOs(os) + '.js'), 'cordova plugin add .*', '\'cordova plugin add ../SalesforceMobileSDK-CordovaPlugin\'');
```

Comment:
if we pass the regexp in a string, the replace will treat it as a string litteral